docs/source/release/v3.14.0/ui.rst

```diff
@@ -11,6 +11,16 @@ UI & Usability Changes
 
 - Added time standard deviation to the sample log dialog
 
+Installation
+------------
+Changes
+#######
+The following changes have been made _only_ on Windows, bringing it in line with Mantid's installation naming on Linux.
+
+- The Mantid Nightly build will now be installed in a different directory by default, to avoid overwriting the release Mantid installation.
+- The Mantid Nightly build desktop and start menu shortcuts will have the "Nightly" suffix appended, to distinguish from the release Mantid installation.
+- Building a package from a pull request has the "Unstable" suffix on the install directory and shortcuts.
```

Comment:
The two points above are great but I think this last one is a little developer centric so can we remove that one?

Reply:
We don't have anything other than annoucing on slack or in a review meeting at the moment.